### DISCUSSION TITLE
How can Zeus improve your ML workloads & pipelines on Kubernetes? Share your pain points and experiences!

### DISCUSSION THREAD

**Discussion by ctrl-alt-lulz** [hpcaitech/ColossalAI#3585] [Development | Developer Experience]
Hey ColossalAI community! 🚀

We're working on an exciting project called Zeus, which aims to simplify and streamline the management of machine learning (ML) workloads and pipelines on Kubernetes for developers of all experience levels. We understand that managing cloud infrastructure can be daunting, especially for those who are new to the world of Kubernetes.

Zeus aims to address this by providing an intuitive UI and natural code interface that allows you to build, maintain, and operate state-of-the-art infrastructure on Kubernetes with ease. Additionally, Zeus helps you optimize your cloud spend through smarter resource scheduling and management. 

GitHub: [https://github.com/zeus-fyi/zeus](https://github.com/zeus-fyi/zeus)

We would love to hear from you about your experiences, challenges, and pain points related to ML workloads and pipelines on Kubernetes! 🧠

Here are some questions to kick off the discussion:

1. What challenges have you faced when deploying and managing ML workloads and pipelines on Kubernetes?

2. How has your team's developer experience been impacted by the complexities of Kubernetes infrastructure management?

3. Are there any specific features or functionalities you would like to see in a solution like Zeus to make your life easier?

Your valuable insights will help us shape the future of Zeus and create a more enjoyable and efficient developer experience for everyone in the ColossalAI community. 🙌

Looking forward to hearing your thoughts! 💡

**Comment by binmakeswell** [comment]
Thanks for sharing; we will consider it in our later schedule.
